js_modules/dagit/packages/core/jest.config.js

```diff
@@ -148,10 +148,7 @@ module.exports = {
   // testLocationInResults: false,
 
   // The glob patterns Jest uses to detect test files
-  // testMatch: [
-  //   "**/__tests__/**/*.[jt]s?(x)",
-  //   "**/?(*.)+(spec|test).[tj]s?(x)"
-  // ],
+  testMatch: ['**/__tests__/**/*.test.[jt]s?(x)'],
```

Comment:
This enforces that only tests in `__tests__` directories will be run, which hopefully serves as a stick to ensure that we're consistent about where we put test files and how we name them.

Reply:
Yeah, same for stories -- let's do this in a followup